assembly/assembly-root-war/src/main/webapp/_app/keycloackLoader.js

```diff
@@ -17,17 +17,6 @@ export class KeycloakLoader {
         const msg = "Cannot load keycloak settings. This is normal for single-user mode.";
```

Comment:
wondering, are we still supporting the single-user mode?

Reply:
As I remember we do not support it

Reply:
so then why do we need this message?